frontend/src/components/Address.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import { GeoapifyGeocoderAutocomplete, GeoapifyContext } from '@geoapify/react-geocoder-autocomplete'
+import '@geoapify/geocoder-autocomplete/styles/minimal.css'
+
+const Address = () => {
+
+  const geoapifyKey = process.env.REACT_APP_GEOAPIFY_KEY
+
+  function onPlaceSelect(value) {
+    console.log(value);
+  }
+
+  function onSuggectionChange(value) {
+    console.log(value);
+  }
+
+    function preprocessHook(value) {
+    return `${value}, Hamburg, Germany`
+  }
```

Comment:
Just a little indent. Btw seems like prettier didn't apply here. If you don't have it installed within your ide, I would suggest you add it. It saves you a lot of time you would have to spend to format manually, since you can set it up to format on save it is quite handy. Can be configured for a single project as well to streamline the prettier configs for your project team, but thats just fyi.
```suggestion
  function preprocessHook(value) {
    return `${value}, Hamburg, Germany`
  }
```

Reply:
I have it, but I see that sometimes it doesn't apply, specially in files with JSX, I don't know why, I will dig in